src/NUnitFramework/tests/Assertions/ConditionAssertTests.cs

```diff
@@ -130,8 +130,8 @@ public void IsNaNFails()
         public void IsEmpty()
         {
             Assert.IsEmpty( "", "Failed on empty String" );
-            Assert.IsEmpty( new int[0], "Failed on empty Array" );
-            Assert.IsEmpty((IEnumerable)new int[0], "Failed on empty IEnumerable");
+            Assert.IsEmpty(Array.Empty<int>(), "Failed on empty Array" );
+            Assert.IsEmpty((IEnumerable)Array.Empty<int>(), "Failed on empty IEnumerable");
```

Comment:
The intent of the test would make the below a better version:
```suggestion
            Assert.IsEmpty(Enumerable.Empty<int>(), "Failed on empty IEnumerable");
```

Reply:
Done